resources/jscomposition/system/table/cell/truncatedGroupOptionsCell/TruncatedGroupOptionsCell.vue

```diff
@@ -1,24 +1,24 @@
 <template>
   <div class="tw-flex tw-relative tw-text-nowrap tw-whitespace-nowrap tw-p-3">
     <div
-      v-if="optionsModel.length && optionsModel[0]?.options"
+      v-if="firstOptionValue"
```

Comment:
please considerate add testing properties to the main components
example 
`data-test="first-option"`